network-api/networkapi/wagtailcustomization/templates/wagtailredirects/index.html

```diff
@@ -0,0 +1,78 @@
+        {% trans "Add redirect" as add_str %}
+        {% url "wagtailredirects:start_import" as import_link %}
+        {% trans "Import redirects" as import_str %}
+
+        <header class="hasform">
+            {% block breadcrumb %}{% endblock %}
+            <div class="row nice-padding">
+                <div class="left">
+                    <div class="col header-title">
+                        <h1 class="icon icon-redirect">{{ redirects_str }}</h1>
+                    </div>
+                    <form class="col search-form" action="{% url "wagtailredirects:index" %}{% if query_parameters %}?{{ query_parameters }}{% endif %}" method="get" novalidate role="search">
+                        <ul class="fields">
+                            {% for field in search_form %}
+                                {% include "wagtailadmin/shared/field_as_li.html" with field=field field_classes="field-small iconfield" input_classes="icon-search" %}
+                            {% endfor %}
+                            <li class="submit visuallyhidden"><input type="submit" value="Search" class="button" /></li>
+                        </ul>
+                    </form>
+                </div>
+                <div class="right has-multiple-actions">
+                    <div class="actionbutton">
+                        <a href="{{ add_link }}" class="button bicolor button--icon">{% icon name="plus" wrapped=1 %}{{ add_str }}</a>
+                    </div>
+                    <div class="actionbutton">
+                        <a href="{{ import_link }}" class="button bicolor button--icon">{% icon name="doc-full-inverse" wrapped=1 %}{{ import_str }}</a>
+                    </div>
+                </div>
+            </div>
+        </header>
+    {% else %}
+        {% include "wagtailadmin/shared/header.html" with title=redirects_str icon="redirect" search_url="wagtailredirects:index" %}
+    {% endif %}
+
+    <div class="nice-padding">
+        <div id="redirects-explanation" class="">
+            <h1>
+                How to use Redirects
+            </h1>
+            <p>
+                Wagtail Redirects are rules for catching 404 errors, also known as "page not found" errors, and instead directing
+                users to another page, or external website. Redirects in the Mozilla Foundation CMS are locale-agnostic, meaning
+                that a redirect for any non-existent url <code>/some-page</code> will trigger regardless of whether it was requested
+                with a locale prefix. A rule for <code>/some-page</code> will automatically kick in for <code>/en/some-page</code>,
+                <code>/fr/some-page</code> etc. Also, note that redirects only work for pages that are <strong>not live</strong>.
+                You cannot create a redirect for a published page: you will have to unpublish that page before a redirect will work.
+            </p>
+            <hr/>
+        </div>
```

Comment:
This is the only new code, the rest is a copy of https://github.com/wagtail/wagtail/blob/cf38b2457dba521d029d3a6e5024a7d0c3f5faf6/wagtail/contrib/redirects/templates/wagtailredirects/index.html